plugins/colorschemes/catppuccin.nix

```diff
@@ -1,12 +1,15 @@
 {
   lib,
-  helpers,
   config,
   pkgs,
   ...
 }:
 with lib;
```

Comment:
(All?) the files still have `with lib`

Reply:
IDK if you'd rather keep that out of scope for this PR though?

Reply:
I can add another commit for that